src/lib/seam/components/DeviceTable/DeviceRow.tsx

```diff
@@ -25,22 +68,24 @@ export function DeviceRow({
   device,
   onClick,
 }: DeviceRowProps): JSX.Element | null {
+  const opacityClass =
```

Comment:
We should use `classNames` below and avoid the empty string. Then I think we can do like this

```
const opacityClassName = accountIsOffline(device) || deviceIsOffline(device) ? 'seam-offline-fade' ? undefined
```

and below

```
      <TableCell className={classNames('seam-table-cell', opacityClassName)}>

```